larreco/RecoAlg/TCAlg/Utils.cxx

```diff
@@ -2544,6 +2705,28 @@ namespace tca {
     return atan2(dw, dt);
   } // TwoTPAngle
 
+  ////////////////////////////////////////////////
+  std::vector<unsigned int> PutHitsInVector(TCSlice& slc, PFPStruct const& pfp, HitStatus_t hitRequest)
```

Comment:
This is the perfect kind of function: it receives arguments, doesn't appear to modify them, and then creates an object and returns it.  Such a function is naturally thread-safe.  Is it possible to mark `slc` as `TCSlice const&` as `slc` doesn't appear to be modified?